apps/prairielearn/src/lib/db-types.ts

```diff
@@ -491,6 +491,24 @@ export const FileEditSchema = z.object({
 });
 export type FileEdit = z.infer<typeof FileEditSchema>;
 
+export const AiGenerationPromptSchema = z.object({
+  completion: z.any(),
+  context: z.string().nullable(),
+  errors: z.array(z.string()),
+  html: z.string().nullable(),
+  id: z.string(),
+  prompt_type: z.enum(['initial_prompt', 'human_revision', 'autorevision']),
+  prompting_user_id: z.string(),
+  python: z.string().nullable(),
+  question_id: z.string(),
+  response: z.string(),
+  title: z.string().nullable(),
+  user_prompt: z.string(),
+  uuid: z.string().nullable(),
```

Comment:
Question: why do we track `title` and `uuid` at all? This question is probably related to the fact that you have "TODO" values for these when inserting into this table.

Reply:
Per conversation a couple weeks ago, was going to adjust prompting to return json schema with title + uuid. Turns out that just blindly switching over hurts accuracy a little, so I might add this in later but not in this PR. Removed in bf2b176c4cffe17e4fedd2434fef03ae520111c9.